spec/support/repo/lib/database.rb

```diff
@@ -0,0 +1,27 @@
+# typed: true
+# frozen_string_literal: true
+
+# We need this workaround otherwise the `todo_spec` thinks
+# Rake::DSL is a todo symbol
```

Comment:
I think you can put the definition here: https://github.com/Shopify/tapioca/blob/main/spec/support/repo/config/shim.rbi

Reply:
Nice! I didn't know it was there.